libs/qCC_db/ccColorTypes.h

```diff
@@ -215,21 +215,30 @@ namespace ccColor
 	};
 
 	//! Conversion from Rgbf
-	inline Rgb FromRgbf(const Rgbf& color)
+	inline Rgb FromRgbfToRgb(const Rgbf& color)
 	{
 		return Rgb( static_cast<ColorCompType>(color.r * MAX),
 					static_cast<ColorCompType>(color.g * MAX),
 					static_cast<ColorCompType>(color.b * MAX) );
 	}
 	
-	//! Conversion from Rgbaf
-	inline Rgb FromRgbf(const Rgbaf& color)
+	//! Conversion from Rgbaf to Rgb
+	inline Rgb FromRgbafToRgb(const Rgbaf& color)
```

Comment:
I was going to suggest a change to this function name when I saw it in the ChaiScript side of things since it hadn't followed the surrounding naming convention, I had renamed it to FromRgbaf